ubuntu/ubuntu20.04/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:20.04
+
+RUN : \
+ && set -exu \
+ && apt update\
```

Comment:
`apt` does not provide a stable API for scripting

Reply:
Did you check if this actually does anything, is the base image updated?

Reply:
I guess not. The version before and after the update is 
`OpenSSL 1.1.1f  31 Mar 2020` 
As I understand the CVE https://www.openssl.org/news/secadv/20220705.txt is patched in v1.1.1f
https://launchpad.net/ubuntu/+source/openssl/1.1.1f-1ubuntu2.16 

So does this mean that there is no need to update?

Reply:
@lietu

Reply:
https://computingforgeeks.com/check-version-of-installed-package-on-ubuntu-debian/

Reply:
I checked the version with `dpkg` and I end up with the same result. 

```
# dpkg -s openssl | grep Version                             
Version: 1.1.1f-1ubuntu2.16
# 
```